### ISSUE TITLE
Flow support

### ISSUE THREAD

**Issue by rattrayalex-stripe** [grpc/grpc-web#340]
Congrats on GA!

I don't expect this to be a short-term priority, but I'd like to track support for [flowtype](https://flow.org/) in addition to TypeScript. 

Is this something that you're considering?

**Comment by stanley-cheung** [comment]
You are right, not a short-term priority. We are still working on getting better typings for TypeScript at the moment. But this can be a longer term goal.

**Comment by rattrayalex-stripe** [comment]
Awesome, thanks for the quick response. Sounds like the right prioritization - best of luck! 

Once TypeScript is stable it should be straightforward to add Flow support; I'd expect that [this](https://github.com/grpc/grpc-web/blob/master/javascript/net/grpc/web/grpc_generator.cc#L504) should mostly work with minimal modifications. When the time comes, you may find [this](https://github.com/niieani/typescript-vs-flowtype#differences-in-syntax) enumeration of the deltas between the langs to be helpful.

**Comment by goodmind** [comment]
Why there is no support for custom generators, like in swagger-codegen and openapi-generator?

**Comment by wenbozhu** [comment]
@goodmind  could you file a separate "feature request" on this, assuming we are talking about openapi/swagger specifically here?

**Comment by goodmind** [comment]
@wenbozhu no, I mean support custom generators without recompiling everything and hardcoding

**Comment by kisszhangxiansen** [comment]
stream.on('data', (res) => {
    this.isLoad = true;
});
Vue parameter  “isLoad” no immediate effect ,I don't know if this is a problem
